container/worker/run_openqa_worker.sh

```diff
@@ -9,7 +9,7 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" --non-unique kvm
```

Comment:
Any chance we can test this? With e.g. a `healthcheck` or something else?

Reply:
I added two checks. First to checks that the kvm group don't have already the id group 0. And an other after the groupmod execution to check hat the id is changed.

However, I think to test this should be done on the openQA tests

Reply:
A possible option is to run `kvm -S daemonize` that creates a vm without execution. If the user doesn't have permissions this command exits with code 1, other ways creates the VM in background. However that needs the installation of some additional packages.

Reply:
If this is going to be an openQA test, I suggest we get this PR merged now. My question was assuming a trivial `healthcheck` or one-line check within `run_openqa_worker.sh`. And I take your answer to mean that's not feasible.

Reply:
So I guess this is what you were adding to *run_openqa_worker.sh*:

    qemu-kvm -S &
    kill $!

Mentioning it here to keep the conversation context. I'm not fond of this approach personally as it requires additional deps only used by this check.